build/images/flow-aggregator/Dockerfile

```diff
@@ -31,4 +31,9 @@ LABEL description="The docker image for the flow aggregator"
 COPY --from=flow-aggregator-build /antrea/bin/flow-aggregator /
 COPY --from=flow-aggregator-build /antrea/bin/antctl /usr/local/bin/
 
+# install ca-certificates
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates \
+    && update-ca-certificates
```

Comment:
best practice is to include ` rm -rf /var/lib/apt/lists/*` to avoid growing the size of the image too much after installing packages

see https://github.com/antrea-io/antrea/blob/dbaebc7a82d32a0270b84f02af0070cdc86276d8/build/images/base/Dockerfile#L54